spec/features/idv/steps/in_person/address_spec.rb

```diff
@@ -39,5 +39,27 @@
       click_on t('idv.cancel.actions.keep_going')
       expect(page).to have_current_path(idv_in_person_proofing_address_url)
     end
+
+    it 'allows user to submit valid inputs on form', allow_browser_log: true do
+      user = user_with_2fa
+      complete_idv_steps_before_address(user)
+
+      # fill in form
+      fill_out_address_form_ok
+
+      # submit
+      click_idv_continue
+      # ssn step
+      expect(page).to have_current_path(idv_in_person_ssn_url, wait: 10)
+      complete_ssn_step(user)
+
+      # verify step has valid inputs from form
```

Comment:
IMHO you could get away with removing these comments; I think your test is easy to read and clear in its intent.